src/main/java/com/volmit/iris/Iris.java

```diff
@@ -443,40 +443,38 @@ public void imsg(CommandSender s, String msg) {
 
     @Override
     public ChunkGenerator getDefaultWorldGenerator(String worldName, String id) {
-        String dimension = IrisSettings.get().getGenerator().getDefaultWorldType();
```

Comment:
Remove all changes from iris, this isnt related to commands, and this change was already merged in.

Reply:
I'll merge master into this again after #546 is resolved.
The only reason I added this here so it can actually be compiled and tested, Ill make sure not to include any changes compared to master in the Iris class near the end.

Reply:
Merged just now so you can do that.

Reply:
Merged remote tracking